tests/payments.py

```diff
@@ -77,11 +77,10 @@ def test_complete_order(self):
         test_payment = Payment.objects.get(pk=self.payment.id)
         self.assertEqual(test_order.payment_type, test_payment)
 
+    def test_delete_paymenttype(self):
+        url = "/paymenttypes/1"
         
-        
-
-        
-
-        
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
+        response = self.client.delete(url)
 
-    # TODO: Delete payment type
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
```

Comment:
Good start but after the delete method is called how do you know it's actually deleted? Trying to get the payment type after it's deleted should give you an error

Reply:
Added additional GET to make sure it was deleted